proof_gen/src/lib.rs

```diff
@@ -74,9 +74,26 @@
 //! ) -> ProofGenResult<GeneratedAggProof> { ... }
 //! ```
 //!
-//! ### Block proofs
+//! ### Transaction Aggregation proofs
 //!
 //! Once the prover has obtained a `GeneratedAggProof` corresponding to the
+//! entire set of segment proofs within one transaction proof, they can then
```

Comment:
"they" refers to the proofs?

Reply:
To the prover, rather. Would it be clearer if we wrote instead: 
```suggestion
//! Given a `GeneratedAggProof` corresponding to the
//! entire set of segment proofs within one transaction proof, the prover can
```